server/util/status/status.go

```diff
@@ -157,10 +157,22 @@ func UnauthenticatedErrorf(format string, a ...interface{}) error {
 
 // Wrap adds additional context to an error, preserving the underlying status code.
 func WrapError(err error, msg string) error {
-	return status.Error(status.Code(err), fmt.Sprintf("%s: %s", msg, err))
+	return status.Error(status.Code(err), fmt.Sprintf("%s: %s", msg, Message(err)))
 }
 
 // Wrapf is the "Printf" version of `Wrap`.
 func WrapErrorf(err error, format string, a ...interface{}) error {
 	return WrapError(err, fmt.Sprintf(format, a...))
 }
+
+// Message extracts the error message from a given error, which for gRPC errors
+// is just the "desc" part of the error.
+func Message(err error) string {
```

Comment:
earmarked this change into the PR since I noticed the runner was printing errors that would look like `rpc code = ... desc = rpc code = ... desc = ...` on successive calls to `Wrap`. lmk if you'd rather see it as a separate PR

Reply:
Not currently -- made it private